src/services.rs

```diff
@@ -134,13 +134,14 @@ impl ChannelKind {
     /// Get the type of values used to communicate with this service.
     pub fn get_type(&self) -> Type {
         use self::ChannelKind::*;
-        use values::Type::*;
+        use values::Type as ValueType;
```

Comment:
No strong opinion here, but I'd tend to keep `Type` because that's the name used in the return value.